pkgs/development/tools/build-managers/qbs/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, qt5, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "qbs-${version}";
+
+  version = "1.8";
+
+  src = fetchFromGitHub {
+    owner = "qt-labs";
+    repo = "qbs";
+    rev = "fa9c21d6908e0dad805113f570ac883c1dc5067a";
+    sha256 = "1manriz75rav1vldkk829yk1la9md4m872l5ykl9m982i9801d9g";
+  };
+
+  enableParallelBuilding = true;
+
+  buildInputs = with qt5; [
+    qtbase
+    qtscript
+  ];
+
+  installFlags = [ "INSTALL_ROOT=$(out)" ];
```

Comment:
@0xABAB `${out}` would refer to the nix value `out` which doesn't exist. Here `$(out)` expands the `out` environment variable, present at build time.